### PR TITLE
Roles for configuring Orchard Workers

### PR DESCRIPTION
**Pull request by fkorotkov**
Roles for configuring Orchard Workers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
+*.iml
```
Comment (edigaryev) [inline .gitignore]:
```suggestion
*.iml

```

File: ansible.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[ssh_connection]
+ssh_args = -o StrictHostKeyChecking=accept-new
```
Comment (edigaryev) [inline ansible.cfg]: I don't think this is secure and fits everyone's threat model. Perhaps we should rather leave this to the user to configure.

File: roles/orchard-worker-service/templates/org.cirruslabs.orchard.worker.plist.j2 [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <key>PATH</key>
+      <string>/bin:/usr/bin:/usr/local/bin:/opt/homebrew/bin</string>
+    </dict>
+    <key>WorkingDirectory</key>
+    <string>/var/empty</string>
+    <key>RunAtLoad</key>
+    <true/>
+    <key>KeepAlive</key>
+    <true/>
+    <key>LaunchOnlyOnce</key>
```
Comment (edigaryev) [inline roles/orchard-worker-service/templates/org.cirruslabs.orchard.worker.plist.j2]: Why do we need this?

File: production-pool [hunk 4]
```diff
@@ -0,0 +1,10 @@
+macs:
+  children:
+    workers:
+      hosts:
+        192.168.192.121:
```
Comment (edigaryev) [inline production-pool]: Perhaps we should leave a more generic value here, similarly to `# populate me` below? E.g. `example.com`.

File: ansible.cfg (hunk 2 above)
Comment (fkorotkov) [reply ansible.cfg]: SGTM. See [5a60d2d](https://github.com/cirruslabs/ansible-orchard/pull/1/commits/5a60d2d83ce4373f7d626e4b46fb84bec1a69b22)

File: roles/orchard-worker-service/templates/org.cirruslabs.orchard.worker.plist.j2 (hunk 3 above)
Comment (fkorotkov) [reply roles/orchard-worker-service/templates/org.cirruslabs.orchard.worker.plist.j2]: Just copied from how we run Persistent Worker. Removed in [e39193e](https://github.com/cirruslabs/ansible-orchard/pull/1/commits/e39193e6cab33471286930598dc03967b042c878)

File: production-pool (hunk 4 above)
Comment (fkorotkov) [reply production-pool]: See [7cca43b](https://github.com/cirruslabs/ansible-orchard/pull/1/commits/7cca43b262d1ba935671dc82f8f142cc1d588d1f) 👌

**Review by edigaryev — APPROVED** [review]

**Pull request merged**